config/config.go

```diff
@@ -674,6 +682,15 @@ type QueueConfig struct {
 	MaxBackoff model.Duration `yaml:"max_backoff,omitempty"`
 }
 
+// MetadataConfig is the configuration for sending metadata to remote
+// storage.
+type MetadataConfig struct {
+	// Send controls whenever we send metric metadata to remote storage.
```

Comment:
I think you meant `controls whether we send`?

Reply:
This doesn't appear to be resolved.